Babylon/commands/powerbi/dataset/services/powerbi_api_svc.py

```diff
@@ -59,6 +63,7 @@ def update_credentials(self, workspace_id: str, dataset_id: str):
         access_token = self.powerbi_token
         response = oauth_request(get_url, access_token)
         if response is None:
+            logger.error(f"[powerbi] failed to update dataset with dataset_id : {dataset_id}")
```

Comment:
`logger.error(f"[powerbi] failed to  update credentials on dataset_id : {dataset_id}")`